controllers/warp/warp.go

```diff
@@ -34,13 +33,25 @@ type Watcher struct {
 	eventRecorder   record.EventRecorder
 }
 
+type eventRecorder interface {
```

Comment:
🚫 **[checkstyle]** <deadcode> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>`eventRecorder` is unused